src/index.js

```diff
@@ -1,11 +1,28 @@
+/* eslint-disable */
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import './index.css';
 import App from './App';
+import Calculator from './components/Calculator';
+import Navbar from './components/Navbar';
+import Home from './components/Home';
+import Quote from './components/Quote';
```

Comment:
- [ ] Please change the file names from Navbar, Home and Quote to navbar, home and quote. You saved those file names with all the letters in lowercase. When I run your site, I get an error saying files were not found because they were miswritten. They should look similar to this
```
import Navbar from './components/navbar';
import Home from './components/home';
import Quote from './components/quote';
```